<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="ImageView" name="imageView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="ImageView" name="imageView_1_edit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Image 1</string>
     </property>
    </widget>
    <widget class="QComboBox" name="image1_FT_component">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Image Components</string>
     </property>
     <item>
      <property name="text">
       <string>Choose FT Components</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Real Component</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Imaginary Component</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="ImageView" name="imageView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Image 2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="image2_FT_component">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Image Components</string>
     </property>
     <item>
      <property name="text">
       <string>Choose FT Components</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Real Component</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FT Imaginary Component</string>
      </property>
     </item>
    </widget>
    <widget class="ImageView" name="imageView_2_edit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Mixer Output To:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="mixer_viewer">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Output1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Output2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Component 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Component 2</string>
     </property>
    </widget>
    <widget class="QSlider" name="FT_slider_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="FT_slider_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="mixer_comp1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imaginary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Phase</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="mixer_image1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Image Components</string>
     </property>
     <item>
      <property name="text">
       <string>Image1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Image2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="mixer_image2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Image Components</string>
     </property>
     <item>
      <property name="text">
       <string>Image1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Image2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="mixer_comp2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>271</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Phase</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Real</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imaginary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Magnitude</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Uniform Phase</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="ImageView" name="output_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="ImageView" name="output_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ourput 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Output 2</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Window"/>
   </widget>
   <widget class="QMenu" name="menuBrowse">
    <property name="title">
     <string>Browse</string>
    </property>
    <addaction name="actionImage1"/>
    <addaction name="actionImage2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBrowse"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImage1">
   <property name="text">
    <string>Image1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionImage2">
   <property name="text">
    <string>Image2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>FT_slider_1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_8</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>876</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FT_slider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_9</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>875</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
